src/components/FormAlertWithSubmitButton.tsx

```diff
@@ -79,10 +80,11 @@ function FormAlertWithSubmitButton({
 }: FormAlertWithSubmitButtonProps) {
     const styles = useThemeStyles();
     const style = [!footerContent ? {} : styles.mb3, buttonStyles];
+    const extraSafePaddingBottomStyle = useSafePaddingBottomStyle();
```

Comment:
To match the hook name. 
```suggestion
    const safePaddingBottomStyle = useSafePaddingBottomStyle();
```